ruby/neo4j/driver/access_mode.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+# Used by Routing Driver to decide if a transaction should be routed to a write server or a read server in a cluster.
+# When running a transaction, a write transaction requires a server that supports writes.
+# A read transaction, on the other hand, requires a server that supports read operations.
+# This classification is key for routing driver to route transactions to a cluster correctly.
+#
+# While any {@link AccessMode} will be ignored while running transactions via a driver towards a single server.
+# As the single server serves both read and write operations at the same time.
+module Neo4j::Driver::AccessMode
+  # Use this for transactions that requires a read server in a cluster
+  READ = Java::OrgNeo4jDriver::AccessMode::READ # TO DO: replace this with :read once all java refs are done
+
+  # Use this for transactions that requires a write server in a cluster
+  WRITE = Java::OrgNeo4jDriver::AccessMode::WRITE # :write # TO DO: replace this with :write once all java refs are done
```

Comment:
to take it further we decided with @Kinju011 to convert all dependent classes. The code needs to load successfully but not necessarily execute. Such a conversion will help us get rid of any java sources and only depend on netty and reactive jars, although I did not plan to release reactive at first release at all, now that you converted it we can keep it. But let's don't have any dependency on `Java::OrgNeo4jDriver` in converted code.

Reply:
we decided as well to keep java dependencies in package notation rather than ruby notation to easily spot the places

Reply:
There are couple of places in code and in test where `Neo4j::Driver::AccessMode::READ/WRITE` are used ([example](https://github.com/neo4jrb/neo4j-ruby-driver/blob/ec429228d3eb3efea4d2c25af7f6c791d9cbb3e1/spec/neo4j/driver/dev_manual_examples_spec.rb#L122)). 

those places were passing java enum in java function call making the test pass. Since I introduced this module that code started breaking as it is now passing ruby symbol instead java enum.

I guess I should remove this module and not migrate it till we have removed such dependencies.

Reply:
I didn't realize that we already have Neo4j::Driver::AccessMode through https://github.com/neo4jrb/neo4j-ruby-driver/blob/f647af00dad736602e2622feb25f689ef71f5957/ruby/neo4j/driver.rb#L18
All guess those `include_package` will cause more problems. I'll try to tackle those conflicts.

Reply:
Maybe rather than removing the module, convert it to its final state and comment out the conflicting constants.

Reply:
when I keep this module with constants commented I get `uninitialized constant Neo4j::Driver::AccessMode::READ`. so removing it.